src/tests/EdFi.DataManagementService.Tests.E2E/Features/Descriptors/DescriptorsValidation.feature.disabled

```diff
@@ -0,0 +1,285 @@
+Feature: Descriptors are a set of mechanisms to support flexible enumerations or code sets
+
+        Background:
+            Given the system has this "educationOrganizationCategoryDescriptors"
+                  | codeValue | description | namespace                                               | shortDescription |
+                  | School    | School      | uri://ed-fi.org/EducationOrganizationCategoryDescriptor | School           |
+              And the system has this "disabilityDescriptors"
+                  | codeValue       | description          | namespace                            | shortDescription     |
+                  | Disability Test | User Disability Test | uri://ed-fi.org/DisabilityDescriptor | User Disability Test |
+
+        @ignore
+        Scenario: 01 / Ensure clients can create a Descriptor with the required attributes
```

Comment:
Adding a slash into the name is something new. Just curious, why add the slash? Should we do that elsewhere?

Reply:
Removed. It was a last minute change and I didn't double check with other files